hadoop-ozone/dist/src/shell/conf/dn-audit-log4j2.properties

```diff
@@ -77,6 +77,14 @@ appender.rolling.policies.time.type=TimeBasedTriggeringPolicy
 appender.rolling.policies.time.interval=86400
 appender.rolling.policies.size.type=SizeBasedTriggeringPolicy
 appender.rolling.policies.size.size=64MB
+appender.rolling.strategy.type=DefaultRolloverStrategy
```

Comment:
Can you add comments on the change similar to the comments above? Same for other changes as well.

Reply:
@aswinshakil Thanks for the review. Comments added.